conf/openmetadata.yaml

```diff
@@ -180,6 +180,7 @@ authenticationConfiguration:
   # This will only be valid when provider type specified is customOidc
   providerName: ${CUSTOM_OIDC_AUTHENTICATION_PROVIDER_NAME:-""}
   publicKeyUrls: ${AUTHENTICATION_PUBLIC_KEYS:-[http://localhost:8585/api/v1/system/config/jwks]}
+  tokenValidationAlgorithm: ${AUTHENTICATION_TOKEN_VALIDATION_ALGORITHM:-"RS256"}
```

Comment:
we need to update the collate config as well, otherwise those builds will fail when we merge this

Reply:
https://github.com/open-metadata/openmetadata-collate/pull/1023 here @harshach